frontend/src/pages/HomePage.tsx

```diff
@@ -14,6 +14,7 @@ import { useEffect, useState } from 'react';
 import { requestMSAuthResult } from '../functions/requestMSAuthResult';
 import { useNavigate } from 'react-router-dom';
 import { loginUser } from '../api';
+import {persistToken} from '../localStorage'
```

Comment:
This line isn't formatted like all the other lines of code. This repo should be set up so that on file save it should automatically format the file. Is that working for you?